random.js

```diff
@@ -0,0 +1,76 @@
+var slideIndex = 0;
+var slides = document.getElementsByClassName("mySlides");
+var dots = document.getElementsByClassName("slide-inactive");
+var i;
+var userClick = false;
+let myTimeout;
+
+function currentSlide(n){
+    showSlides(slideIndex = n);
```

Comment:
Not sure this function wrapper is necessary.

Reply:
Also I'm not sure if `showSlides(slideIndex = n);` does what you think it does

Reply:
You are right about showSlides, wasn't doing what I thought it was.

Reply:
:)